File: cartracker/src/main/java/ch/rasc/cartracker/entity/option/Make.java
package ch.rasc.cartracker.entity.option;

import javax.persistence.Entity;

import ch.rasc.extclassgenerator.Model;

@Entity
@Model(value = "CarTracker.model.option.Make", readMethod = "makeOptionService.read",
		createMethod = "makeOptionService.create",
		updateMethod = "makeOptionService.update",
		destroyMethod = "makeOptionService.destroy", paging = true)
public class Make extends BaseOption {
	// nothing special
}

File: cartracker/src/main/webapp/app/model/option/Make.js
Ext.define("CarTracker.model.option.Make",
{
  extend : "Ext.data.Model",
  fields : [ {
    name : "longName",
    type : "string"
  }, {
    name : "shortName",
    type : "string"
  }, {
    name : "createDate",
    type : "date",
    dateFormat : "c"
  }, {
    name : "active",
    type : "boolean",
    defaultValue : true
  }, {
    name : "id",
    type : "int",
    useNull : true,
    convert : null
  } ],
  proxy : {
    type : "direct",
    api : {
      read : "makeOptionService.read",
      create : "makeOptionService.create",
      update : "makeOptionService.update",
      destroy : "makeOptionService.destroy"
    },
    reader : {
      root : "records"
    }
  }
});